### DISCUSSION TITLE
求一份ChatGPT的分流规则

### DISCUSSION THREAD

**Discussion by koo17173** [blackmatrix7/ios_rule_script#1378] [Q&A] [answered]
没找到ChatGPT的分流规则，有知道的朋友吗

**Comment by ROCK88888888** [comment]
> 朋友

同求

**Reply by koo17173** [reply, marked as answer]
https://github.com/blackmatrix7/ios_rule_script/blob/master/rule/Shadowrocket/OpenAI/OpenAI.list
